File: CS/Solution5.Module.Win/WinModule.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Win.Templates;

namespace Solution5.Module.Win
{
    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class Solution5WindowsFormsModule : ModuleBase
    {
        public Solution5WindowsFormsModule()
        {
            InitializeComponent();
            BarActionItemsFactoryProvider.Instance = new MyBarActionItemsFactoryProvider();
        }
    }
}

File: VB/Solution5.Module.Win/WinModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.ComponentModel

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Win.Templates

Namespace Solution5.Module.Win
	<ToolboxItemFilter("Xaf.Platform.Win")> _
	Public NotInheritable Partial Class Solution5WindowsFormsModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
			BarActionItemsFactoryProvider.Instance = New MyBarActionItemsFactoryProvider()
		End Sub
	End Class
End Namespace
